core/src/test/java/org/elasticsearch/search/searchafter/SearchAfterIT.java

```diff
@@ -102,7 +103,7 @@ public void testsShouldFail() throws Exception {
                     .addSort("field1", SortOrder.ASC)
                     .setQuery(matchAllQuery())
                     .searchAfter(new Object[]{1})
-                    .get();
+                    .execute().actionGet();
```

Comment:
I see `.get()` more than `.execute().actionGet()` in newer tests and I use it myself just because its less typing. I'm just curious why you changed.